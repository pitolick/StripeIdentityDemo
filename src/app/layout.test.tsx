import { render } from '@testing-library/react'
import RootLayout, { metadata } from './layout'

describe('RootLayout', () => {
  it('正しくレンダリングされること', () => {
    const { asFragment } = render(<RootLayout>テストコンテンツ</RootLayout>)
    expect(asFragment()).toMatchSnapshot()
  })
})

describe('metadata', () => {
  it('必要なプロパティが存在すること', () => {
    expect(metadata).toHaveProperty('title')
    expect(metadata).toHaveProperty('description')
  })
})
