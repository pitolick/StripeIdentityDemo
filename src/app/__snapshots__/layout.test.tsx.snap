// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RootLayout 正しくレンダリングされること 1`] = `
<DocumentFragment>
  テストコンテンツ
</DocumentFragment>
`;
